src/System.Runtime/tests/System/Text/StringBuilderTests.cs

```diff
@@ -1436,12 +1473,30 @@ public static void Remove_Invalid()
             var builder = new StringBuilder("Hello");
             Assert.Throws<ArgumentOutOfRangeException>("startIndex", () => builder.Remove(-1, 0)); // Start index < 0
             Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(0, -1)); // Length < 0
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "Parameter name changes in error messages are breaking, not currently backported.")]
+        public static void Remove_Index_Invalid()
+        {
+            var builder = new StringBuilder("Hello");
 
             Assert.Throws<ArgumentOutOfRangeException>("index", () => builder.Remove(6, 0)); // Start index + length > 0
```

Comment:
It looks like that's simply because they aren't being used yet, according to this https://github.com/dotnet/corefx/pull/11074#discussion-diff-76515196

Do you want to try uncommenting?